pkg/providers/vsphere/validator.go

```diff
@@ -476,15 +475,15 @@ func (v *Validator) validatePrivs(ctx context.Context, privObjs []PrivAssociatio
 	var err error
 	missingPrivs := []missingPriv{}
 	passed := false
+	username := vsc.Username()
 
 	for _, obj := range privObjs {
 		path := obj.path
 		privsContent := obj.privsContent
 		t := obj.objectType
-		username := vsc.Username()
 		privs, err = v.getMissingPrivs(ctx, vsc, path, t, privsContent, username)
 		if err != nil {
-			return passed, err
+			return passed, fmt.Errorf("failed to get missing privs: %v", err)
```

Comment:
If this error is something we return to the users, should we be more explicit and say privileges?
```suggestion
			return passed, fmt.Errorf("failed to get missing privileges: %v", err)
```

Reply:
ah yes!